tagging_audio_effects/tools/codebook_generator.py

```diff
@@ -0,0 +1,23 @@
+"""
+This file will take the class map of YaMNet and generate a codebook for RedHen's reference.
+"""
+import csv
+
+if __name__ == '__main__':
+    YAMNET_CLASS_MAP_FILE = "../models/assets/yamnet_class_map.csv"
+    OUTPUT_FILE = "../codebook/codebook_yamnet_1.0.csv"
+    print("Generating codebook from ", YAMNET_CLASS_MAP_FILE)
+    input_file = open(YAMNET_CLASS_MAP_FILE, 'r')
+    output_file = open(OUTPUT_FILE, 'w')
+    data = csv.reader(input_file)
```

Comment:
Are you sure you want to use csv.reader and writer?

Ex: https://stackoverflow.com/a/60561297 ( for writer ) ... not required, but as shared, pandas is good to know [ and handy/often-quicker to use ]

Reply:
I wasn't trying to convert a comma delimiter to a pipe, rather keep it in csv but only change the ones with tag.
Like from (Chirp, tweet) to Chirp | tweet. 
I know about pandas and used in other tools here as well, but as it was so simple use case I did not use it. If you say I will use pandas.

Reply:
Pandas is not strict requirement